src/sensor/ping360.h

```diff
@@ -543,7 +543,7 @@ class Ping360 : public PingSensor
     int _angular_speed = 1;
     bool _autoTransmitDuration = true;
     uint _central_angle = 1;
-    bool _configuring = true;
+    bool _configuring = false;
```

Comment:
It's working here, how is it broken ? There is an issue for that open ? Is this PR breaking the autobaud logic ?